### ISSUE TITLE
NetSUS won't apply my update selections

### ISSUE THREAD

**Issue by krusej23** [jamf/NetSUS#54]
Running NetSUS 3.0.2 with Ubuntu 14.04.1 on NetSUS appliance.

I can checkmark an update in our production catalog and when I click apply the checkmark is gone.

**Comment by justinrummel** [comment]
By checkmark are you referring to the Software Update Server section?

**Comment by krusej23** [comment]
Yes, none of the updates at the top of the screen will allow me to apply them.
![screen shot 2015-02-03 at 4 17 23 pm](https://cloud.githubusercontent.com/assets/10549869/6030862/29eabb06-abc0-11e4-8b54-5995aa33e4a2.png)

**Comment by krusej23** [comment]
Any solutions for this problem? We are stuck with not being able to push out updates without this working. We are being told be JAMF to downgrade to Ubuntu 12 something and I would like to stay away from having to redo all of this.

**Comment by justinrummel** [comment]
Essential this us Reposado.  I would start reviewing the options on their github repo at: https://github.com/wdas/reposado/tree/master/docs and see what you can do via command line vs. trusting the GUI.  Not sure where in NetSUS's GUI system is not working correctly.

**Comment by kev111** [comment]
I had this problem too, and this fixed it for me:

I noticed this in the /etc/httpd/logs/ssl_error_log:

PHP Warning: Unknown: Input variables exceeded 1000. To increase the limit change max_input_vars in php.ini. in Unknown on line 0

So I changed /etc/php.ini and set max_input_vars = 10000

After that I can checkmark all of the updates.


I also posted this solution here: https://jamfnation.jamfsoftware.com/discussion.html?id=14461
